.php-cs-fixer.dist.php

```diff
@@ -27,7 +26,7 @@
         "php_unit_fqcn_annotation" => true,
         "phpdoc_summary" => true,
         "phpdoc_types" => true,
-        "psr4" => true,
+        "psr_autoloading" => true,
         "return_type_declaration" => ["space_before" => "none"],
         "short_scalar_cast" => true,
         "single_blank_line_before_namespace" => true,
```

Comment:
This should use our shared config in `amphp/php-cs-fixer-config:^2`, see https://github.com/amphp/amp/blob/d048ec1d03d47fc313d630e989b7a73053f10fae/.php-cs-fixer.dist.php

Reply:
Be that as it may, I do not think this issue needs to hold up this PR, nor even the 1.0 release, since styles can just be fixed after the fact.